mesa/space.py

```diff
@@ -243,7 +250,7 @@ def get_neighbors(self, pos, moore,
         return list(self.iter_neighbors(
             pos, moore, include_center, radius))
 
-    def torus_adj(self, pos):
+    def torus_adj(self, pos: Coordinate) -> Tuple[int, int]:
```

Comment:
Return type should be Coordinate as well

Reply:
Fixed.